promtimer/promtimer.py

```diff
@@ -252,7 +287,57 @@ def main():
     grafana_port = args.grafana_port
     prometheus_base_port = grafana_port + 1
 
-    if not args.cluster:
+    cbbackupmgr_stats_mode = False
```

Comment:
This section is too long to be added inline. In fact, suggest adding a new `backupstats` module and adding this code there.